### PR TITLE
s2u-48: Show student items based on groups

### PR DESCRIPTION
**Pull request by frasese**
- Fixed initial collapse section bug

### CODE REVIEW

File: microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<div th:if="${typesKeys == null or typesKeys.empty}" class="sak-banner-info" th:text="#{no_synchronized}"></div>
+			
+<th:block th:if="${typesKeys != null and !typesKeys.empty}">
+	<div class="container-view container-fluid collapse" th:classappend="${!treeView}? 'in'">
+		<div class="container-header">
+			<h2><span th:text="#{tree_view}"/><span role="button" class="icon-toggle fa fa-toggle-off" onclick="toggleView()" th:title="#{tree_view_off}"></span></h2>
```
[the remark this answers is not in the document]
Comment (frasese) [reply microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html:6]: it has a title

File: microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+							th:selected="${sortBy} == ${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_BY_DATE} + ${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_ASCENDING}" 
+							th:text="#{sort_by_date_ascending}"></option>
+						<option 
+							th:value="${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_BY_DATE} + ${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_DESCENDING}" 
+							th:selected="${sortBy} == ${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_BY_DATE} + ${T(org.sakaiproject.microsoft.mediagallery.controller.MainController).SORT_DESCENDING}" 
+							th:text="#{sort_by_date_descending}"></option>
+					</select>
+				</div>
+				<div>
+					<input
```
[the remark this answers is not in the document]
Comment (frasese) [reply microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html:30]: it has a title

File: microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+						th:placeholder="|&#xF002; #{search}|"
+					/>
+				</div>
+			</form>
+		</div>
+		<div class="container-fluid">
+			<div class="panel-group" id="accordion_all">
+				<div class="panel panel-default" th:each="typeKey, stat : ${typesKeys}" th:with="itemsList=${allItemsByType.get(typeKey)}">
+					<div class="panel-heading">
+						<h2 class="panel-title section-title">
```
[the remark this answers is not in the document]
Comment (frasese) [reply microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/body.html:44]: it's not empty

**Comment by mpellicer** [comment]
This looks great to me, anything you want to mention before merge it?

**Comment by frasese** [comment]
Rebased to match last force-push to s2u-22x

**Comment by mpellicer** [comment]
I'm fine with all of this, let me know when do you want to merge.

**Comment by frasese** [comment]
Would be great if this is merged as soon as posible, because s2u-50 uses some of these features.

**Review by mpellicer — APPROVED** [review]
Great job @frasese ! You're a monster

**Pull request merged**